### ISSUE TITLE
focus on tools instead of image

### ISSUE THREAD

**Issue by mowgli** [BestImageViewer/geeqie#49]
i personally like geeqie very much, but one thing that annoys me is that when i double-click an image to open it in geeqie the keyboard focus is automatically set to the pathbar in the tools part of the window. instead it should be on the image (at least that's my opinion) so that you can immediately use shortcuts like "+" to zoom in or "page_down" to go to the next image.

(i'm using alpha 3 on xubuntu 8.10)

Reported by: *anonymous
